spikeextractors/baseextractor.py

```diff
@@ -274,6 +274,9 @@ def load_extractor_from_pickle(pkl_file):
             extractor._features = d['features']
         return extractor
 
+    def get_unsorted_spike_train(self):
```

Comment:
@ferchaure can you move this to the SortingExtractor class? This base extractor is a common class for recording and sorting extractors which handles allocating memory, dumping etc

Reply:
ok, sorry
should I include a _start_frame_ and _end_frame_ inputs?

Reply:
That would be perfect!